### PR TITLE
Updated to include Library Carpentry

### PR DESCRIPTION
**Pull request by libcce**
Updated to include Library Carpentry per recent Carpentries/LC merger agreements and @tracykteal, John Chodacki.

### CODE REVIEW

**Comment by ChristinaLK** [comment]
@libcce there's an open PR that substantially modifies this page that I'd like to merge first, so this may require some fiddling after I merge it.  

@tracykteal (or @weaverbel?) can you confirm that this accurately reflects the current relationship b/n LC and the Carpentries as a whole?

**Comment by weaverbel** [comment]
@libcce @ChristinaLK @tracykteal 
Re this in Chris's PR:
In May, 2018, The Carpentries Executive Council and the Library Carpentry Governance Group agreed to begin the process of aligning operations and Library Carpentry as a Lesson Organization. All  organizations will continue to maintain their own curricular materials.

@libcce I would reword it as follows:
In early 2018, The Carpentries Executive Council and the Library Carpentry Governance Group agreed to open discussions about bringing Library Carpentry under The Carpentries umbrella as a distinct lesson organization. Initially, while talks continue, The Carpentries will assist Library Carpentry with workshop requests and instructor training. Questions of governance, mission and vision, and other questions about aligning organizations will take place over the coming months.

**Comment by lexnederbragt** [comment]
We have landed on the term 'Lesson Project', rather than organisation (as many non-profits have Projects as main activities, and to reflect that SWC and DC no longer are independent organisations). Could that be taken into account here too?

**Comment by tracykteal** [comment]
Thanks for putting this in @libcce. We do want to include information about Library Carpentry in instructor training, so people know the current status and some background on the content and motivation of those lessons. I think @weaverbel language accurately represents the current state, and we can update to use 'Lesson Project' rather than 'Lesson Organization' as @lexnederbragt suggests. Given the merge conflicts with @ChristinaLK  PR, I would suggest closing this PR and putting in a new one with the updated suggested language.

**Comment by libcce** [comment]
Thank you all for your suggestions. I will close this PR and incorporate your feedback into another PR.
